template/build/common.props

```diff
@@ -1,12 +1,15 @@
 <Project>
   <Import Project="..\version.props" />
-  <Import Project="..\.build\common.props" Condition="Exists('..\.build\common.props')" />
 
-  <Target Name="EnsureInitialized"
-          BeforeTargets="Build"
-          Condition="!Exists('$(MSBuildThisFileDirectory)..\.build\common.props')">
-    <Error File="$(MSBuildProjectFile)"
-           Text="Project has not been initialized. Run 'build initialize' in the solution directory." />
-  </Target>
+  <ItemGroup>
+    <PackageReference Include="Internal.AspNetCore.Sdk" Version="1.0.0-*" PrivateAssets="All" />
```

Comment:
NB: this file is an example of the changes we'll make in all the aspnet repos that are currently on VS 2017.